released/packages/coq-monae/coq-monae.0.4.2/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/monae"
+dev-repo: "git+https://github.com/affeldt-aist/monae.git"
+bug-reports: "https://github.com/affeldt-aist/monae/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "Monads and equational reasoning in Coq"
+description: """
+This Coq library contains a hierarchy of monads with their laws used
+in several examples of monadic equational reasoning."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install_full"]
+depends: [
+  "coq" { (>= "8.14" & < "8.16~") | (= "dev") }
```

Comment:
Recall that Coq 8.16.0 is due to be released any day now, and if monae supports MC 1.15.0 it likely supports 8.16.0.
```suggestion
  "coq" { (>= "8.14" & < "8.17~") | (= "dev") }
```